containers/codespaces-linux/definition-manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-	"definitionVersion": "0.16.1",
+	"definitionVersion": "0.17.0",
```

Comment:
FYI, @dmgardiner25 is going to upgrade `gh` CLI to 1.0.0, so let's not ship this until the GH change is made as well.

Reply:
Actually, good news on that one. The script in kitchen sink should be automatically grabbing the latest release, so that should be done.  

I just noticed the old env var is getting passed in still, but its not set, which results in the latest coming down. I'll remove that bit.

![image](https://user-images.githubusercontent.com/312252/93627636-13c24500-f99a-11ea-96ae-d16ccd9768d0.png)

Reply:
Ah, cool, I missed that!